test/correctness/simd_op_check_x86.cpp

```diff
@@ -638,14 +644,17 @@ int main(int argc, char **argv) {
         {
             Target("x86-32-linux"),
             Target("x86-32-linux-sse41"),
-            Target("x86-64-linux-sse41-avx"),
-            Target("x86-64-linux-sse41-avx-avx2"),
+            // Always turn on f16c when using avx. Haswell had avx without f16c,
```

Comment:
Actually, Haswell had F16C (together with newly-added AVX2+FMA). Even IvyBridge supported F16C (without AVX2 or FMA). Only SandyBridge was pure AVX.

Reply:
Oops, I had haswell and ivy bridge in the wrong order in my head. I'll fix the comment.